content/case_histories/mt_isa/properties.rst

```diff
@@ -43,7 +43,7 @@ Overall we expect any mineralization unit in this area to be **conductive and ch
 +---------------------------+-------------------+------------------------------------------+--------------------------+
 | Surprise Creek Formation  |  Low              | High (~1000)                             |  None                    |
 +---------------------------+-------------------+------------------------------------------+--------------------------+
-| Eastern Creek Volcanics   |  Low              | High (~1000)                             |  High                    |
+| Eastern Creek Volcanics   |  Low              | High (~1000)                             |  Low                     |
```

Comment:
@fourndo : can you confirm? 
(also do we want the title to be **Chargeability** instead of **Induced Polarization**